docs-new/docs/components/certificate-operator.md

```diff
@@ -1,65 +1,23 @@
-# Keptn Certificate Manager
+# Keptn Cert Manager
 
-## Keptn Cert Manager
-
-The Keptn Cert Manager automatically configures TLS certificates to
+The Keptn Cert Manager is a Kubernetes operator that
+automatically configures TLS certificates to
 [secure communication with the Kubernetes API](https://kubernetes.io/docs/concepts/security/controlling-access/#transport-security).
-You can instead
-[use cert-manager.io](../installation/configuration/cert-manager.md)
-for this purpose.
 
-Keptn includes a Mutating Webhook
-that requires TLS certificates to be mounted as a volume in its pod.
-In version 0.6.0 and later, the certificate creation
-is handled automatically by
-the [keptn-cert-manager](https://github.com/keptn/lifecycle-toolkit/blob/main/keptn-cert-manager/README.md).
+Keptn includes a Mutating, Validating and Conversion Webhooks
```

Comment:
```suggestion
Keptn includes a mutating, validating and conversion webhook
```
OR
```suggestion
Keptn includes mutating, validating and conversion webhooks
```
Singular/plural issue -- is this one webhook with three functions or three webhooks?  And we've decided to not capitalize "webhook" so I figure the modifiers should not be capitalized, either